ads/README.md

```diff
@@ -130,6 +130,22 @@ Here are some factors that affect whether the resize will be executed:
 - Whether the resize is requested for a currently active ad;
 - Whether the resize is requested for an ad below the viewport or above the viewport.
 
+
+### Support for variable size ad requests
+In order to support multi-size ad requests, AMP accepts an optional `data` param to `window.context.renderStart` which will automatically invoke request resize with the width and height passed.
+In case the resize is not successful, AMP will horizontally and vertically center align the creative within the space initially reserved for the creative.
+This utility makes it convenient for ad networks to support mult-size ad requests in AMP.
+#### Example
+```javascript
+window.context.renderStart(data);
```

Comment:
@jasti That is correct; I'm talking about ad units that start off with one size, and then expand to a different size at a later stage in their lifecycle (mostly because of user initiated interaction). We have a few units that do that, and I've noticed similar formats being provided by other platforms as well. Although it might be a little redundant, I think it might be helpful to clarify that one might still need to support the `resizeRequest` API (and all the UX and creative additions that go along with it), to ensure that the format works as expected.